### PR TITLE
[Chore] Update metadata for Runtime 43

### PR DESCRIPTION
**Pull request by ken-centrality**
## Summary

Update metadata for Runtime 43

## Checklist

- [x] Add description

### CODE REVIEW

File: packages/api-types/src/registry/interfaces.ts [hunk 1]
```diff
@@ -1052,22 +1052,22 @@ import type {
 	InstructionV2,
 	InteriorMultiLocation,
 	Junction,
-	Junctions,
-	JunctionsV1,
-	JunctionsV2,
 	JunctionV0,
 	JunctionV1,
 	JunctionV2,
+	Junctions,
```
Comment (KarishmaBothara) [inline packages/api-types/src/registry/interfaces.ts:1058]: does yarn format change these files?

Comment (ken-centrality) [reply packages/api-types/src/registry/interfaces.ts:1058]: yup, and it should be fine I reckon

**Comment by ken-centrality** [comment]
Fixed CI, @KarishmaBothara can you please review again?

**Review by KarishmaBothara — APPROVED** [review]
lgtm

**Pull request merged**